packages/qwik/src/core/use/use-on.ts

```diff
@@ -18,7 +19,7 @@ import { useInvokeContext } from './use-core';
  * @public
  */
 // </docs>
-export const useOn = (event: string | string[], eventQrl: QRL<(ev: Event) => void> | undefined) =>
+export const useOn = (event: CustomEventType, eventQrl: QRL<(ev: Event) => void> | undefined) =>
```

Comment:
`event`'s type should be `CustomEventType | CustomEventType[]`. Same for `useOnDocument` and `useOnWindow`'s `event` parameter.

Reply:
Right I am on it thanks for the feedback!